Book/php7/internal_types/functions/callables.rst

```diff
@@ -0,0 +1,129 @@
+PHP Callables
+===================
+
+Dealing with PHP functions in C requires the knowledge of the following two structures
+``zend_fcall_info``/``zend_fcall_info_cache``. The first one necessarily contains the information for calling
+the function, such as arguments and the return value, but may also include the actual callable.
+The latter *only* contains the callable. We will use the commonly used abbreviation of FCI and FCC when talking about
+``zend_fcall_info`` and ``zend_fcall_info_cache`` respectively.
+You will most likely encounter those when using the ZPP ``f`` argument flag, or when you need to call a PHP function
+or method from within an extension.
+
+Structure of ``zend_fcall_info``
+--------------------------------
+
+A ``zend_fcall_info`` has the following structure::
```

Comment:
Yeah just looked at the def via heap.space, and it changed at lot during the 7.X release cycle.
Will mention that it is only as of 8.0 and not document 7.x mainly because I don't know how some of the other fields work.